src/providers/ipa/ipa_subdomains_id.c

```diff
@@ -253,6 +253,25 @@ static void ipa_subdomain_account_got_override(struct tevent_req *subreq)
         }
     }
 
+    /* If looking up the auto private group for a user with a login name override,
+     * switch from BE_REQ_GROUP to BE_REQ_USER to fetch the user.
+     */
+    if (state->ar->filter_type == BE_FILTER_NAME && dp_error == DP_ERR_OK
+        && state->override_attrs == NULL
+        && ((state->ar->entry_type & BE_REQ_TYPE_MASK) == BE_REQ_GROUP)) {
```

Comment:
Hi,

this is too early. Currently this breaks looking up groups from trusted domains which do not have an id-override. You should apply a similar logic after the lookup of the group itself failed. Additionally you should check if the current domain is a MPG domain or not because only for MPG domains auto-private groups are available.

bye,
Sumit

Reply:
Thank you, I made the requested changes.

Reply:
Changes are now pushed correctly, CI is running